doc/gs_cluster.md

```diff
@@ -33,7 +33,7 @@ Those commands will install TC8 and optimisation for GS like TurboJPEG, Marlin R
 Then open the */etc/default/tomcat8* file and set **JAVA_OPTS** to : 
 
 ```
--Djava.awt.headless=true -XX:+UseConcMarkSweepGC -Xms2G -Xmx2G -XX:PermSize=256m -DGEOSERVER_DATA_DIR=/opt/geoserver_datadir -Dfile.encoding=UTF8 -Djavax.servlet.request.encoding=UTF-8 -Djavax.servlet.response.encoding=UTF-8 -server -XX:+UseParNewGC -XX:ParallelGCThreads=2 -XX:SoftRefLRUPolicyMSPerMB=36000 -XX:NewRatio=2 -XX:+AggressiveOpts -Djava.library.path=/usr/lib/jni:/opt/libjpeg-turbo/lib64
```

Comment:
So we should use : 
```
-XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:ParallelGCThreads=4
```
in the georchestra doc ?

I think we can remove `-XX:NewRatio=2` because it's the default value of the JVM : http://www.oracle.com/technetwork/java/javase/tech/vmoptions-jsp-140102.html

Same for `-XX:+AggressiveOpts` which seems to be default activated on version upper 5.0 update 6

Can you confirm it ?